data/data/openstack/masters/variables.tf

```diff
@@ -45,3 +45,13 @@ variable "server_group_id" {
   type        = string
   description = "ID of the server group to assign the servers to."
 }
+
+variable "additional_network_ids" {
+  type        = list(string)
+  description = "Names of additional network IDs for master nodes."
```

Comment:
`IDs of additional networks for master nodes` sounds more correct.

Reply:
done